packages/relay/src/lib/services/ethService/ethCommonService/index.ts

```diff
@@ -61,6 +61,7 @@ export class CommonService implements ICommonService {
   static blockLatest = 'latest';
   static blockEarliest = 'earliest';
   static blockPending = 'pending';
+  static isDevMode = !!(process.env.DEV_MODE && process.env.DEV_MODE === 'true');
```

Comment:
Why the added complexity on this assignment?

```suggestion
  static isDevMode = process.env.DEV_MODE === 'true';
```